src/tools/build-manifest/src/main.rs

```diff
@@ -71,6 +71,7 @@ static TARGETS: &[&str] = &[
     "arm-unknown-linux-gnueabihf",
     "arm-unknown-linux-musleabi",
     "arm-unknown-linux-musleabihf",
+    "arm64ec-pc-windows-msvc",
```

Comment:
@Mark-Simulacrum Is this enough to add the target to `rustup` or is that configured at a separate location?

Reply:
This should be sufficient: https://rust-lang.zulipchat.com/#narrow/stream/233931-t-compiler.2Fmajor-changes/topic/Promote.20arm64ec-pc-windows-msvc.20to.20tier.202.20compiler-team.23746/near/443353113